msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,26 +176,133 @@ TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest T
 
 Microsoft Teams JavaScript library exposes the frameContext in which your meeting tab URL is loaded in the getContext API. The possible values of frameContext are content, task, setting, remove, sidePanel, and meetingStage. This allows you to build customized experiences based on where the app renders. For example, showing a specific collaboration focused UI when in the `meetingStage` and a different meeting preparation UI in the chat tab (`content`). For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).
 
+## App caching
+
+App caching helps apps to launch faster, which gives a better experience for users to return to the app. App caching improves subsequent launch time of the apps that are loaded in the meeting side panel.
+
+> [!NOTE]
+>
+> * Currently, app caching is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * App caching is supported only for tabs loaded in the meeting side panel in Teams desktop client.
```

Comment:
Change to:
App caching is supported only for tabs loaded in the meeting side panel in Teams desktop client. While it can work in other contexts (ie. personal app, chat/channel tabs), we don't officially support it and encourage app developers to only register onLoad/beforeUnload handlers when in the sidePanel frame context

Reply:
I came here to leave a similar comment, so let's just say "+1" to that :)

Reply:
Please retain phrasing I included. we need to be explicit about this

Reply:
Updated as below:

"App caching is supported only for tabs loaded in the meeting side panel in Teams desktop client. While it can work in other contexts such as personal apps and chat or channel tabs, it isn't officially supported. We recommend to register the `onLoad` or `beforeUnload` handlers when in the sidePanel frameContext."